3-Authorization-II/3-App-CallAPI/README.md

```diff
@@ -0,0 +1,393 @@
+---
+page_type: sample
+name: Enable your Java Spring Boot daemon to sign in users with Azure Active Directory and call a protected Web api
+description: Enable your Java Spring Boot daemon to sign in users in your Azure AD tenant and call a Web api protected with Spring Security and Microsoft Identity Platform
+languages:
+ - java
+products:
+ - azure-active-directory
+ - msal-java
+urlFragment: ms-identity-java-spring-tutorial
+extensions:
+- services: ms-identity
+- platform: Java
+- endpoint: AAD v2.0
+- level: 200
+- client: Java Spring Daemon
+- service: Java Spring Web API
+---
+
+# Enable your Java Spring Boot daemon to sign in users with Azure Active Directory and call a protected Web api
+
+[![Build status](https://identitydivision.visualstudio.com/IDDP/_apis/build/status/AAD%20Samples/.NET%20client%20samples/ASP.NET%20Core%20Web%20App%20tutorial)](https://identitydivision.visualstudio.com/IDDP/_build/latest?definitionId=XXX)
+
+* [Overview](#overview)
+* [Scenario](#scenario)
+* [Prerequisites](#prerequisites)
+* [Setup the sample](#setup-the-sample)
+* [Explore the sample](#explore-the-sample)
+* [Troubleshooting](#troubleshooting)
+* [How to deploy this sample to Azure](#how-to-deploy-this-sample-to-azure)
+* [Next Steps](#next-steps)
+* [Contributing](#contributing)
+* [Learn More](#learn-more)
+
+## Overview
+
+This sample demonstrates a Java Spring Daemon calling a Java Spring Web API that is secured using Azure AD.
+
+## Scenario
+
+This sample demonstrates a Java Spring Daemon calling a Java Spring Web API that is secured using Azure AD.
+
+1. The client Java Spring Daemon uses the [MSAL Java](https://aka.ms/identityplatform) to sign-in a user and obtain a JWT [ID Token](https://aka.ms/id-tokens) and an [Access Token](https://aka.ms/access-tokens) from **Azure AD**.
+1. The **access token** is used as a *bearer* token to authorize the user to call the Java Spring Web API protected by **Azure AD**.
```

Comment:
> The **access token** is used as a *bearer* token to authorize the user to call the Java Spring Web API protected by **Azure AD**.

Codegen should have generated this according to a daemon app (i.e. no mention of user). If that's not happening by default, can we create an issue in the Codegen repo to address this?